docs/extend/modular-resources/examples/triton.md

```diff
@@ -49,7 +55,7 @@ Add the following to your `"modules"` array:
   "type": "registry",
   "name": "viam_mlmodelservice-triton-jetpack",
   "module_id": "viam:mlmodelservice-triton-jetpack",
-  "version": "latest"
+  "version": "0.4.0"
```

Comment:
Let's add some text right below this that says "replace the value of the `version` field with the value you determined above with the `docker images` command".